app/stac_api/collection_temporal_extent.py

```diff
@@ -438,33 +517,29 @@ def update_temporal_extent(
                     item.pk,
                     qs
                 )
+                )
```

Comment:
Code style, indentation issue

Reply:
I'll change that. Strange that linting didn't complain